src/vs/workbench/contrib/chat/browser/actions/chatActions.ts

```diff
@@ -98,14 +99,14 @@ class OpenChatGlobalAction extends Action2 {
 		}
 		if (opts?.previousRequests?.length && chatWidget.viewModel) {
 			for (const { request, response } of opts.previousRequests) {
-				chatService.addCompleteRequest(chatWidget.viewModel.sessionId, request, undefined, 0, { message: response });
+				chatService.addCompleteRequest(chatWidget.viewModel.sessionId, request, variables ? { variables } : undefined, 0, { message: response });
```

Comment:
Why add variables to previous requests? If you want that, shouldn't those variables be on previousRequests?

Reply:
Oh, I misunderstood what was happening here.